### PR TITLE
[bug 1167192] In python, stop using the gettext from tower.

### PR DESCRIPTION
**Pull request by rlr**
As mentioned in IRC, I created fjord/base/translation.py to create the lazy versions. I am open to other ideas.

r?

### CODE REVIEW

File: fjord/base/translation.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.utils.functional import lazy
+from django.utils.translation import ugettext, ungettext
+
+
+ugettext_lazy = lazy(ugettext, unicode)
+ungettext_lazy = lazy(ungettext, unicode)
```
Comment (willkg) [inline fjord/base/translation.py]: It looks like django has lazy versions already:

https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#lazy-translation

Does that not work for what we're doing?

Comment (rlr) [inline fjord/base/translation.py]: oooh. thanks for looking into that! I'll use that

**Comment by rlr** [comment]
fixord ^^

**Comment by willkg** [comment]
I ran the tests and they're fine. (No surprise there.)

I went through the feedback in French and didn't see any missing strings.

That's probably good enough for this. r+

**Pull request merged**